cmd/cdk/cdk.go

```diff
@@ -237,6 +243,7 @@ func mustGetRPCClient(ctx context.Context, rpcURL string) *ethclient.Client {
 func getRollupManager(cdkArgs parsedCDKArgs, rpcClient *ethclient.Client, addr common.Address) (*rollupManager, *abi.ABI, error) {
 	var contract *rollupManager
 	var contractABI *abi.ABI
+	log.Info().Msgf("Getting rollup manager contract at address %s", addr.Hex())
```

Comment:
I would prefer to avoid using stuff like `Msgf` in favor of

`log.Info().Stringer("rollupMangerAddress", addr).Msg("Getting rollup manager")`